src/website/src/app/documentation/demos/datagrid/filtering/filtering.html

```diff
@@ -81,6 +81,17 @@ <h2>Custom filters</h2>
 <clr-code-snippet [clrCode]="examples.colorFilterTS" clrLanguage="typescript"></clr-code-snippet>
 <clr-code-snippet [clrCode]="examples.colorFilterHTML" clrLanguage="html"></clr-code-snippet>
 
+<p>
+    When you apply the filter to columns that use some visual transformation like <code class="clr-code">Pipes</code> or manually apply information by concatenating strings, 
```

Comment:
`By default, filtering searches the original model value for matches. In cases where you format the text for display (such as using a pipe), you may want to create a custom filter to handle searching the formatted text. Otherwise, the results you see may not be filtered in the way you expect.`